### PR TITLE
Nk/kasli soc

### PR DESCRIPTION
**Pull request by nkrackow**

### CODE REVIEW

File: sinara.py [hunk 1]
```diff
@@ -82,7 +83,8 @@ class Sinara(_SinaraTuple):
         "Clocker": "2x4 clock fan out",
         "Booster": "8x RF power amp",
         "Booster_Channel": "RFPA module",
-        "DIO_MCX": "16x MCX IO"
+        "DIO_MCX": "16x MCX IO",
+        "Kasli_soc": "12 EEM ZYNQ SoC"
```
Comment (jordens) [inline sinara.py:88]:
```suggestion
        "Kasli_SoC": "12 EEM ZYNQ SoC"
```

File: sinara.py [hunk 2]
```diff
@@ -60,7 +60,8 @@ class Sinara(_SinaraTuple):
         "Clocker",
         "Booster",
         "Booster_Channel",
-        "DIO_MCX"
+        "DIO_MCX",
+        "Kasli_soc"
```
Comment (jordens) [inline sinara.py:65]:
```suggestion
        "Kasli_SoC"
```

File: deploy_sinara.py [hunk 3]
```diff
@@ -89,8 +89,8 @@ def flash(description, ss, ft_serial=None):
                     ee = EEPROM(bus)
                     if i == 0:
                         port = "LOC0"
-                        if si.hw_rev in ["v2.0",]:
-                            ee = EEPROM(bus, addr=0x57)  # v2
+                        if si.hw_rev in ["v2.0",] or si.board_fmt in ["Kasli_soc"]:
```
Comment (jordens) [inline deploy_sinara.py]: Then write this as

```python
if (si.board_fmt, si.hw_rev) in [
  ("Kasli", "v2.0"),
  ("Kasli_SoC", "v1.0"),
  # ...
]:
```

**Review by jordens — CHANGES_REQUESTED** [review]

File: sinara.py (hunk 2 above)
Comment (nkrackow) [reply sinara.py:65]: This comes from the .json like this. I used the exactly the same naming there as the m-labs template. Not sure if `Kasli_SoC` in the .json would work for building the firmware..

File: sinara.py (hunk 1 above)
Comment (jordens) [reply sinara.py:88]: https://github.com/quartiq/kasli-i2c/pull/11#discussion_r930763928

**Review by jordens — APPROVED** [review]

**Pull request merged**